Gruntfile.js

```diff
@@ -104,8 +104,8 @@ module.exports = function(grunt) {
 
         watch: {
             all: {
-                files: ['Gruntfile.js', 'src/**/*'],
-                tasks: ['jshint']
+                files: ['Gruntfile.js', 'src/**/*', 'test/**/*', 'assets/js/**/*'],
```

Comment:
Can you remove `assets/js/**/*` from the files array as it's supposed to be more of a build type of folder. I'm going to make it so there aren't 2 taggle.js source files being edited with every commit.